lib/MetaCPAN/Util.pm

```diff
@@ -18,7 +18,6 @@ use Ref::Util      qw(
     is_plain_hashref
     is_ref
 );
-use Cpanel::JSON::XS ();
```

Comment:
This removal is wrong. There are subs manually imported from this package later in the file.